Rakefile

```diff
@@ -292,16 +271,6 @@ namespace :install do
     end
   end
 
-  task :shared do
-    if local_environment?
-      Dir.chdir(shared_js_dir) do
-        shared_js_build = CDO.use_my_shared_js ? shared_js_dir('build/package') : 'shared-package'
-        RakeUtils.ln_s shared_js_build, dashboard_dir('public','shared')
```

Comment:
Should this change include some kind of update script to remove shared-package and/or any symlinks from existing developer workstations?